app/lib/navigation/state_file_nc_question_navigation.rb

```diff
@@ -27,7 +27,7 @@ class StateFileNcQuestionNavigation < Navigation::StateFileBaseQuestionNavigatio
         Navigation::NavigationStep.new(StateFile::Questions::WaitingToLoadDataController),
       ]),
       Navigation::NavigationSection.new("state_file.navigation.section_5", [
-        Navigation::NavigationStep.new(StateFile::Questions::PostDataTransferController, false),
+        Navigation::NavigationStep.new(StateFile::Questions::PostDataTransferController),
```

Comment:
we should also change this for NJ unless we were explicitly told not to show post-data transfer page for NJ (`state_file_nj_question_navigation`) I don't think NY matters in the grand scheme of things, but noting that changes in the navigation was added for all states here https://github.com/codeforamerica/vita-min/pull/5058/files